tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrationUtil.java

```diff
@@ -108,6 +110,18 @@ public static IResource getIResourceFromFile( File f )
         return retval;
     }
 
+    public static IResource getResourceFromMigrationProblems( MigrationProblems problem )
+    {
+        String projectName = problem.getSuffix();
+        IProject project = CoreUtil.getWorkspaceRoot().getProject( projectName );
+
+        if( project.exists() )
+        {
+            return project;
+        }
```

Comment:
always add newline before return statement unless its the only line in the body of a method.